tests/README.md

```diff
@@ -74,6 +153,35 @@ Container-based tests:
 - `--docker`: Uses Docker as container engine. Accepts the base image name.
 - `--container`: Uses Podman as container engine. Accepts the base image name.
 
+To use container backends, use the following options on the command line:
+
+- Podman - `--container <base image name>`
+- Docker - `--docker <base image name>`
+
```

Comment:
This is kind of duplicate of the previous paragraph. Could you consolidate the 2 paragraphs together?

Reply:
Consolidated.